src/main/java/org/myrobotlab/service/OpenCV.java

```diff
@@ -455,7 +455,7 @@ static public ServiceType getMetaData() {
     meta.addDependency("com.github.axet", "vget", "1.1.34");
 
     // yolo models
-    meta.addDependency("yolo", "yolov2", "v2", "zip");
+    meta.addDependency("yolo", "yolov2", "v3", "zip");

```

Comment:
When did we start supporting yolo v3?!  Was that added recently?  I didn't think that was supported in the current version of opencv...

Reply:
changing to "yolo", "yolov2", "0.0.2"